quickwit-cli/src/index.rs

```diff
@@ -703,22 +706,19 @@ pub async fn merge_or_demux_cli(
     let source_config = SourceConfig {
         source_id: "void-source".to_string(),
         source_type: "void".to_string(),
-        params: json!(null),
+        params: json!({}),
     };
     run_index_checklist(&args.metastore_uri, &args.index_id, Some(&source_config)).await?;
     let metastore_uri_resolver = MetastoreUriResolver::default();
     let metastore = metastore_uri_resolver.resolve(&args.metastore_uri).await?;
-    let mut index_metadata = metastore.index_metadata(&args.index_id).await?;
-    let storage_uri_resolver = quickwit_storage_uri_resolver();
-    let storage = storage_uri_resolver.resolve(&index_metadata.index_uri)?;
-    index_metadata.sources = vec![source_config];
     let indexer_config = IndexerConfig {
         data_dir_path: args.data_dir_path,
         ..Default::default()
     };
-    index_metadata.indexing_settings.demux_enabled = demux_enabled;
-    index_metadata.indexing_settings.merge_enabled = merge_enabled;
-    index_data(index_metadata, indexer_config, metastore, storage).await?;
+    // FIXME
```

Comment:
Supporting this is actually going to suck. I'm going to have to propagate those settings all the way down.

Reply:
indeed, I would not use the indexing server in this case, it's annoying but I think it's acceptable for now. I guess we will have some code to launch jobs later.

Reply:
As discussed offline, I will implement a dedicated command for this use case.